application/views/template.php

```diff
@@ -48,16 +48,16 @@
 							$services_down = '';
 
 							// Check LMD.
-							exec('service lmd status', $output, $return_var_lmd);
+							exec('/usr/bin/systemctl status lmd.service', $output, $return_var_lmd);
 							// Check Livestatus.
-							exec('service naemon status', $output, $return_var_le);
+							exec('/usr/bin/systemctl status naemon.service', $output, $return_var_le);
 
-							if($return_var_lmd !== 0 && $return_var_le !== 0){
+							if($return_var_lmd === 3 && $return_var_le === 3){
```

Comment:
Why not revert to !== 0 if the issue on error 127 is cleared up?
From the exit codes non zero seems to indicate that the service is not running

Reply:
to fit to the else statement, when a different scenario happens. if the services were running but might have encountered an error.